source/Company.py

```diff
@@ -1,53 +1,184 @@
+# Company Class exercise
+
+def contains_number(value):
+    '''
+    A function which checks if a value includes digits
+    '''
+    if True in [char.isdigit() for char in value]:
+        return True
+    return False
+
+
+def check_string(str: str):
+    '''
+    A function which checks if the conditions of
+    company name are met
+    '''
+    if contains_number(str) or len(str) < 2:
+        return False
+    elif str[0].isupper() and len(str) >= 2:
+        return True
+
+
+def check_int(val):
+    '''
+    A function which checks if a given number is int
+    '''
+    if isinstance(val, int):
+        return True
+    return False
```

Comment:
This function is not needed. it can be written like this:
```python
def check_int(val):
    return isinstance(val, int)
```
So... to make it short, just use ```isinstance(val, int)```

Reply:
what happens if val is not an integer?

Reply:
This is a simple question you can figure out by yourself. It will return False

Reply:
Ok. Even when I did mention False in the code at all?